sgx-tools/README.md

```diff
@@ -31,6 +31,7 @@
 
 - golang 1.14 or above.
 - `sgx-tools` depends on [SGX DCAP](https://github.com/intel/SGXDataCenterAttestationPrimitives). Please download and install the `libsgx-dcap-quote-verify-devel` rpm (centos) or `libsgx-dcap-quote-verify-dev` deb (ubuntu) from [website](https://download.01.org/intel-sgx/sgx-linux/2.13.3/).
+- `libsgx-aesm-quote-ex-plugin`.
 
```

Comment:
Please add the install information about `libsgx-aesm-quote-ex-plugin`  which is installed in `https://download.01.org/intel-sgx/sgx-linux/2.13.3/` as well.

Reply:
Change to:
```
## Dependency

- Golang 1.14 or above.
- libsgx-dcap-quote-verify-devel (CentOS) / libsgx-dcap-quote-verify-dev (Ubuntu) from [official Intel SGX DCAP release page](https://01.org/intel-software-guard-extensions/downloads).
- libsgx-aesm-quote-ex-plugin from [official Intel SGX Linux release page](https://01.org/intel-software-guard-extensions/downloads).
```

Reply:
> Change to:
> 
> ```
> ## Dependency
> 
> - Golang 1.14 or above.
> - libsgx-dcap-quote-verify-devel (CentOS) / libsgx-dcap-quote-verify-dev (Ubuntu) from [official Intel SGX DCAP release page](https://01.org/intel-software-guard-extensions/downloads).
> - libsgx-aesm-quote-ex-plugin from [official Intel SGX Linux release page](https://01.org/intel-software-guard-extensions/downloads).
> ```

Done, thanks.